apps/webhook/src/health/health.controller.ts

```diff
@@ -0,0 +1,30 @@
+import { Controller, Get } from '@nestjs/common';
+import { HealthCheck, HealthCheckService, HttpHealthIndicator } from '@nestjs/terminus';
+import { DalService } from '@novu/dal';
+import { version } from '../../package.json';
+
+@Controller('v1/health-check')
```

Comment:
So this is just code I have moved from api to application generic and would not like to do fixes in it right now for this PR, because it is already massive. @scopsy might be able to answer this...